src/Coveo.Connectors.EasyFilePusher/Program.cs

```diff
@@ -232,6 +232,17 @@ private static string GetPushApiUrl(ProgramArguments p_Args)
                         default:
                             throw new InvalidEnumArgumentException(INVALID_CLOUD_ENVIRONMENT);
                     }
+                case CloudRegion.ApSouthEast2:
+                    switch (p_Args.environment) {
+                        case CloudEnvironment.Prod:
+                            return Constants.Endpoint.ApSoutheast2.PROD_PUSH_API_URL;
+                        case CloudEnvironment.Hipaa:
+                        case CloudEnvironment.QA:
```

Comment:
What happens if the environment is Hipaa or QA? It throws InvalidEnumArgumentException?

Reply:
Yes!  The effect of not including code between 2 (or more) `case`s is that the series share the same code than the last value.  In this case, it throws the `InvalidEnumArgumentException`.